app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ConfirmationsController < Devise::ConfirmationsController
+  def show
+    if params[:confirmation_token].present?
+      @original_token = params[:confirmation_token]
+    elsif params[resource_name].try(:[], :confirmation_token).present?
+      @original_token = params[resource_name][:confirmation_token]
+    end
+
+    self.resource = resource_class.find_by_confirmation_token(
+        Devise.token_generator.digest(self, :confirmation_token, @original_token)
```

Comment:
Line is too long. [81/80]<br>Indent the first parameter one step more than the previous line.